src/translations/en.json

```
@@ -18,5 +18,6 @@
   "widget.install.install.version.tabs.npm.content.title": "Install payment widget into your npm project.",
   "widget.install.install.version.tabs.npm.content.description.1": "It will work only if you have initiated your npm project and have a package.json in the installation directory.",
   "widget.install.install.version.tabs.npm.content.description.2": "You need to have node js version <14 installed on your computer",
-  "widget.install.install.version.tabs.npm.content.copy": "Copy"
+  "widget.install.install.version.tabs.npm.content.copy": "Copy",
```

Why its is like install.install ?
```suggestion
  "widget.install.version.tabs.npm.content.copy": "Copy",
```

---

`widget.install` is the common key name. And `install` is a section inside that page